app/controllers/questions/personal_info_controller.rb

```diff
@@ -19,6 +19,10 @@ def visitor_record
       Intake.find_by(id: session[:intake_id]) || current_intake
     end
 
+    def prev_path
```

Comment:
It's usually a bummer when a controller has to care about it's neighbors by overriding next_path/prev_path

A better way here would be for WelcomeController to declare that it is never shown:
```
def self.show?(_intake, _current_controller)
  false
end
```

...then the normal implementation of `prev_path` for PersonalInfoController would not see it, and return nil

Another alternative is to remove `WelcomeController` from the navigation order completely, but then you'd have to find some other way to preserve its routes in routes.rb.

If it remains in `GyrNavigationOrder`, you might want to put a comment next to it saying it's deprecated / exists only for redirect / will someday be removed.